sktime/forecasting/base/_meta.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, forecasters, n_jobs=None):
 
     def _check_regressor(self):
         if self.regressor is None:
-            self.regressor_ = GradientBoostingRegressor(max_depth=5)
+            self.regressor_ = GradientBoostingRegressor(max_depth=5, random_state=1)
```

Comment:
Shouldn't we pass a random state here which a user can define in the constructor?
```suggestion
            self.regressor_ = GradientBoostingRegressor(max_depth=5, random_state=self.random_state)
```

Reply:
I was not sure on this, I thought if user want to set seed, a `regressor` can be given with the respective params. But let me add a param `random_state ` as a shorthand